ustriage/ustriage.py

```diff
@@ -362,9 +367,36 @@ def main(start=None, end=None, debug=False, open_in_browser=False,
     else:
         activitysubscribers = []
     bugs = create_bug_list(
-        start, end, lpname, bugsubscriber, nodatefilter, activitysubscribers
+        date_range['start'], date_range['end'],
+        lpname, bugsubscriber, nodatefilter, activitysubscribers
     )
-    print_bugs(bugs, open_in_browser, shortlinks, blacklist=blacklist)
+    print_bugs(bugs, open_browser['triage'], shortlinks, blacklist=blacklist)
+
+    if expiration['show_expiration']:
+        logging.info('---')
+        logging.info('Expiration check I')
+        expire = (datetime.now() - timedelta(days=expiration['expire_next']))
+        expire = expire.strftime('%Y-%m-%d')
+        bugs = create_bug_list(OLDESTTRIAGE,
+                               expire,
+                               lpname, TEAMLPNAME, None, None,
+                               tag=["server-next", "-bot-stop-nagging"])
+        logging.info('Bugs tagged %s older than %s',
+                     expiration['tag_next'], expiration['expire_next'])
+        print_bugs(bugs, open_browser['exp'], shortlinks,
+                   blacklist=blacklist)
+
+        logging.info('---')
+        logging.info('Expiration check II')
+        expire = (datetime.now() - timedelta(days=expiration['expire']))
+        expire = expire.strftime('%Y-%m-%d')
+        bugs = create_bug_list(OLDESTTRIAGE,
+                               expire,
+                               lpname, TEAMLPNAME, None, None,
+                               tag="-bot-stop-nagging")
+        logging.info('Bugs older than than %s', expiration['expire'])
+        print_bugs(bugs, open_browser['exp'], shortlinks,
+                   blacklist=blacklist)
 
```

Comment:
I think at the current size we make readability worse by splitting it.
The if to check if they are to be shown groups it quite nicely.

And yes - reusing most of the search code was my intention to keep this reasonable.